csync/src/CMakeLists.txt

```diff
@@ -104,34 +102,22 @@ set_target_properties(
       RUNTIME_OUTPUT_DIRECTORY
         ${BIN_OUTPUT_DIRECTORY}
 )
-if(BUILD_OWNCLOUD_OSX_BUNDLE)
-  INSTALL(
-    TARGETS
-      ${CSYNC_LIBRARY}
-    LIBRARY DESTINATION
-	${LIB_INSTALL_DIR}
-    ARCHIVE DESTINATION
-	${LIB_INSTALL_DIR}
-    RUNTIME DESTINATION
-	${BIN_INSTALL_DIR}
-    )
-else()
-  INSTALL(
+
+INSTALL(
   TARGETS
     ${CSYNC_LIBRARY}
   LIBRARY DESTINATION
-    ${LIB_INSTALL_DIR}/${APPLICATION_EXECUTABLE}
+    ${LIB_INSTALL_DIR}
```

Comment:
This is only valid if the csync_library is now a public lib. Does it have to be? This has effects on our packaging as well.